src/styles/components/ToolTip.scss

```diff
@@ -16,7 +16,9 @@
 
 .tooltip {
     .is-solid {
-        color: $tooltip-icon-color!important;
+        &:hover, &:focus {
+            color: $tooltip-icon-color !important;
```

Comment:
on hover, tooltip icon is changing from grey to blue. It's not looking good.

Reply:
If we remove this color, then the clarity react's blue color will take over and that will be blue but with different shade of it., Initially I removed it but had to add in a different comment just to keep the color as per DD standards